components/DashboardLayout/Sidebar.tsx

```diff
@@ -53,13 +53,20 @@ const SidebarsByRole: Record<UserRoleType, React.ReactNode> = {
   Mentor: (
     <>
       <SidebarLink href="/mentor/players">Players</SidebarLink>
+      <SidebarLink href="/admin/users">Members</SidebarLink>
```

Comment:
the path should match up with what type of user is signed in so it should be `/mentor/users` this might mean that you have to pull users out of the admin folder